metrics_sdk/test/integration/periodic_metric_reader_test.rb

```diff
@@ -0,0 +1,86 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+require 'test_helper'
+
+describe OpenTelemetry::SDK do
+  describe '#periodic_metric_reader' do
+    before { reset_metrics_sdk }
+
+    it 'emits 2 metrics after 10 seconds' do
+      OpenTelemetry::SDK.configure
```

Comment:
What does this set?

Reply:
This will also configure the meter_provider through [patch](https://github.com/open-telemetry/opentelemetry-ruby/blob/main/metrics_sdk/lib/opentelemetry/sdk/metrics/configuration_patch.rb#L25).